devicetypes/smartthings/smartalert-siren.src/smartalert-siren.groovy

```diff
@@ -12,25 +12,44 @@
  *
  *  SmartAlert Siren
  *
- *  Author: SmartThings
- *  Date: 2013-03-05
+ *  Author: Antonio San Roman
+ *  Date: 2019-10-13
  */
 metadata {
 	definition (name: "SmartAlert Siren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
 		capability "Actuator"
 		capability "Switch"
-		capability "Sensor"
 		capability "Alarm"
-		capability "Health Check"
 
-		command "test"
+		command "EstableceModoSirena"
+        command "EstableceModoStrobe"
+        command "EstableceModoBoth"
 
-		fingerprint deviceId: "0x1100", inClusters: "0x26,0x71"
-		fingerprint mfr:"0084", prod:"0313", model:"010B", deviceJoinName: "FortrezZ Siren Strobe Alarm"
+		attribute "modosSirena", "string"	
+
+		//fingerprint deviceId: "0x1100", inClusters: "0x26,0x71"
```

Comment:
We generally don't want to ever remove fingerprints from an existing DTH. Also the `SZ-SRN12N` is a Zigbee device but this DTH is intended for Z-Wave devices and uses the `ZigBee Siren` DTH.